mm2src/coins/z_coin.rs

```diff
@@ -616,6 +625,132 @@ async fn sapling_state_cache_loop(coin: ZCoin) {
     }
 }
 
+struct TransactionShielded {
+    id_out: i32,
```

Comment:
Primary key is automatically incremented, you don’t have to set it explicitly in the code.